src/modules/75-cf/components/FlagActivation/FlagPrerequisites.tsx

```diff
@@ -319,6 +323,7 @@ export const FlagPrerequisites: React.FC<FlagPrerequisitesProps> = props => {
                             intent="primary"
                             text={getString('cf.shared.prerequisites')}
                             icon="small-plus"
+                            data-testid="prerequisites"
```

Comment:
no need to change but I often like to affix what the thing is to the end of the string e.g

```suggestion
data-testid="prerequisites-button"
```

just makes it a bit easier to follow the tests at a glance - just something to consider going forward

Reply:
Totally agree have changed, thank you